index.html

```diff
@@ -0,0 +1,162 @@
+
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Capstone Project 1</title>
+    <link rel="stylesheet" href="style.css">
+</head>
+<body>
+    <nav>
+        <div class="mobile-version">
+            <div class="hamburger"><img src="images/images/mobile_menu 1.png" alt="loading"></div>
+            <div class="logo"><img src="images/images/logo.png" alt=""></div>
+            <div class="links links-inactive">
+                <span class="close">&times</span>
+                <ul>
+                    <li><a href="" class="nav-link">About</a></li>
+                    <li><a href="" class="nav-link">Program</a></li>
+                    <li><a href="" class="nav-link">Join</a></li>
+                    <li><a href="" class="nav-link">Sponsor</a></li>
+                    <li><a href="" class="nav-link">News</a></li>
```

Comment:
- [ ] Looking at the provided design, you can see that the logo is not visible on the mobile screen. According to the requirement, the logo is meant to be available only on desktop screens, I'll suggest you rectify this. :smile: :wave: 

- - Screenshot:
 
![Selection_014](https://github.com/denigit5/First-Capstone/assets/107632780/bfc6ae9a-6a2b-4e5b-99b1-3658338e8a67)

Reply:
As requested by the previous code reviewer this is a required change and it need to be worked on. It would be nice if you corrected this inorder for you to get an approval. Please pay attention to detail